.github/ISSUE_TEMPLATE/system-tests.md

```diff
@@ -18,6 +18,73 @@ The following issue aims to run all `system tests` for the current release candi
 | **Tag**                              |                                            |
 | **Previous system tests issue**      |                                            |
 
+## Instructions
+For running tests in an AWS EC2 virtual environment, it will be needed to meet the following requirements:
+
+| Environment                  | EC2                                       |
+|------------------------------|-------------------------------------------|
+|Basic_cluster                 |Ubuntu 22.04.2 LTS `C5`.`XLarge` `15GB` HD |
+|Big_cluster_40_agents         |Ubuntu 22.04.2 LTS T3.Large `60GB` HD      |
+|Agentless_cluster             |Ubuntu 22.04.2 LTS T3.Large 30GB HD        |
+|Four_manager_disconnected_node|Ubuntu 22.04.2 LTS T3.Large 30GB HD        |
+|One_manager_agent             |Ubuntu 22.04.2 LTS T3.Large 30GB HD        |
+|Manager_agent                 |Ubuntu 22.04.2 LTS T3.Large 30GB HD        |
+|Enrollment_cluster            |Ubuntu 22.04.2 LTS T3.Large 30GB HD        | 
+| Basic_environment            |Ubuntu 22.04.2 LTS T3.Large 30GB HD        |
+
+
+These requirements should be requested from the @cicd-team.
+
+
+For its execution, the installation of various packages is required, which will be detailed below.
+
+<details><summary>Steps</summary>
+
+```
+### Updating dependencies
+sudo apt update
+sudo apt upgrade
+
+### Installing git and cloning wazuh-qa project
```

Comment:
Change `project` to `repository`

Reply:
Done in https://github.com/wazuh/wazuh/pull/19080/commits/c81065dc33d44b5ff2af146a3783d78205b70aa5